DLLModsSource/SonicGenerations/SonicUnleashedHUD/UnleashedHUD/HudTitle.cpp

```diff
@@ -0,0 +1,183 @@
+#include "HudResult.h"
```

Comment:
Remove these includes, they're already in the pre-compiled header.

Reply:
You mean all of them or only HudResult?

Reply:
All of them, see this code review as to why: https://github.com/brianuuu/PTKickass_ModUpdates/pull/1/files#r889574673